pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.poetry]
+name = "medio"
+version = "0.4.2"
```

Comment:
> Very nice 👍 I had two comments. Can you please exclude PR #7 commits from this PR? It will be easier to review

Yeah... I mistakenly rebased my changes on yours and now it's like this... 
If you have any idea how can I clean in up, I would welcome it. I'll look into it tomorrow, in any case.

Reply:
Just needed to rebase again on master